solutions/hw1/task2/bash.py

```diff
@@ -0,0 +1,69 @@
+import os
+
+
+def wc(first: str, *args: str):
+    def wc_for_one(filepath: str) -> dict:
+        size = os.path.getsize(filepath)
+        with open(filepath) as f:
+            words, lines = 0, 0
+            for line in f:
+                lines += 1
+                words += len(line.split())
+        return {"newlines": lines, "words": words, "bytes": size}
+
+    def print_stats(dictionary: dict, f_name: str):
```

Comment:
Theoretically, this function does not care what's inside the dictionary. So here you can write `Dict[Any, Any]`.

Reply:
Or you can use `[str, int]` and then this will serve as an additional check that a correct dictionary was given to the function.